src/util/loanUtils.js

```diff
@@ -136,3 +136,24 @@ export function watchLoanData({
 	// Return the observer to allow modification of variables
 	return queryObserver;
 }
+
+export function watchLoanCardData({
```

Comment:
Was this really needed? Did reusing the existing method fail in some way?

Reply:
The other requires cookies data to add the loans in the basket, for this card that's not necessary.